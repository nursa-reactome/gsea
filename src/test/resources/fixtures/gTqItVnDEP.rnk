AJUBA	2.012062333
AKR1C2	-2.0787986108616208
AKR1C3	-3.646380210980244
ALDH1L2	2.2140840653
ANKH	2.0384534332
ANKRD28	-2.842617530694578
ANXA2	2.0123412834
ANXA8L1	-2.1548090061830076
AOC1	8.6672841319
APOL6	2.0577746512
ARL4C	5.3640214262
ARRDC3	2.396792476
ASNS	2.3537726863
ATP10A	3.7615571248
ATP6V1C2	-2.625150254215639
AXL	11.5519139848
BCAT1	2.0223882687
BCL11A	2.1414307146
BCL2A1	-2.7626637898098263
BEND4	2.0308518509
BEX5	2.1725358744
BIK	-2.0077087519451253
BST2	2.0431427421
C19orf66	2.6639186011
CACHD1	2.4424939692
CADPS2	2.5757543321
CAPN6	2.3613632675
CCDC88A	2.6976347017
CD52	-2.514994072742449
CD53	3.1753486346
CDC42BPA	8.3163184316
CELF2	-2.409476950307512
CGNL1	-2.1924592069993047
CLEC12A	2.2550118269
CMPK2	4.3674312709
CNR1	-2.515682754973935
CSPG4	2.9603648671
CTNNA2	2.0560209998
CXCL10	7.7394712135
CXCL11	3.5431437532
DAPK1	-2.1342955949016758
DDX17	2.0085717526
DDX58	3.3378635978
DDX60	4.1539533085
DEFA4	-2.699430362749582
DEFB1	-2.594758014642369
DOCK8	2.03825563
EGR2	-2.3507807669083083
EGR3	-3.208124209604614
EIF2AK2	2.7820917519
ENAH	2.0892498033
EPHA7	-3.1090956475957507
EPRS	2.1327945515
EPSTI1	14.7961507564
FIGN	-2.504807962602965
FKBP1B	2.1827395582
FOXF2	2.0521269747
FSTL1	2.4918116109
FUT11	-2.1018212790864763
FYN	2.2139536208
GABBR2	2.1358273038
GBP2	2.3958706152
GBP5	2.0958638491
GJA3	2.5100132848
GLDN	-2.3836129652039264
GNLY	-2.227591974252002
GPATCH11	2.0152027557
GPR27	2.7109997873
GPX8	2.032548813
GUCY1B3	-2.019103688164819
HELZ2	2.2878381592
HERC5	6.9066685951
HERC6	4.3649647404
HESX1	2.8795532475
HIP1	2.3019715446
HIPK2	-2.031506527471003
HS6ST2	2.3648029921
IFI16	11.2828920182
IFI27	474.2749957901
IFI35	2.4351151583
IFI44	52.3724067649
IFI44L	169.3612485523
IFI6	19.9241324269
IFIH1	2.0343952519
IFIT1	29.8474339496
IFIT2	12.039428456
IFIT3	11.9216333179
IFIT5	2.8581368588
IFITM1  19.9537085194
IFITM1  11.4
IFITM2	2.8108374004
IFITM3	5.2513923918
IFNB1	2.3345346472
IKZF2	2.0787625883
INHBE	2.5227897907
INSIG1	2.1834658997
IRF7	4.561378851
IRF9	2.4831905976
ISG15	10.4902706867
ISG20	3.0651655581
ITGA6	-2.2302493226985938
ITGA9	3.0548573217
ITM2A	2.458911822
ITPRIPL2	2.0738621484
KCNK1	2.8095810123
KCTD15	2.8522590159
KDELC1	-2.5486353990577992
KIAA1462	-2.160013007131766
KIAA1551	2.5496602211
KLHDC10	-2.0472254679281683
LAP3	2.2259944567
LDLRAD3	2.1805015259
LGALS3BP	6.6870040913
LST1	2.0068043911
LY6E	4.1239304446
LYST	2.1467065627
LYZ	2.3823163455
MAFB	2.9635471275
MAGEB2	3.3388817506
MAP7	3.633777287
MARCKS	-2.154577510332025
MFNG	3.2091917643
MLLT4	2.2774667635
MX1	49.4731892645
MX2	4.2607429011
MYBPH	-2.48079925874512
NASP	2.6280268219
NCAM1	2.0571970643
NEXN	3.7679635268
NKG7	2.2891150913
NUPR1	2.0012341826
OAS1	13.0874307656
OAS2	18.1928542762
OAS3	6.5418559863
OASL	3.3837078236
ODF3B	2.5292506222
ORM1	3.0795616289
PARP14	3.3106167308
PARP9	3.041367177
PAX5	3.9465991654
PCDH7	-2.0961689470113756
PCOLCE2	-2.2714995189772673
PHACTR1	-2.102360390397325
PHGDH	2.0206087499
PI15	-2.0262820428020607
PLBD1	2.1680379345
PLSCR1	2.008710981
PRDM13	2.288924696
PRLR	3.6768746658
PSAT1	2.1487237276
RAP2A	2.2414930094
RNASE6	2.332100562
RNF175	-2.1186128833410867
RNF213	2.7019294407
RPS6KA2	2.1505117292
RSAD2	3.4126205268
S100P	2.6443625716
SAMD9	2.7363731896
SAMD9L	5.1437659532
SEPP1	-2.1115172011257397
SEPT4	2.2213088521
SHISA5	2.0784528203
SIGLEC1	12.1577975886
SLC16A14	2.1053571632
SLC26A6	-2.0658564366428807
SP110	3.3538426395
SPOCK1	2.0164183661
STAP1	2.4665764441
STAT1	2.456407648
SUCNR1	-2.2115916041495667
TCP11L1	-4.835885211583122
TGFB1I1	-2.0945858258516292
TMCC3	2.3961861672
TNF	2.2283409641
TNFSF13B	2.6240223353
TNNT1	-2.1397170036138804
TP53INP2	-2.2093010181889836
TPM1	-2.1872073557568137
TRIM22	55.2462639546
TRIM5	2.6699357038
TRIM69	2.0698769429
TTC39B	2.0906622341
TWIST2	2.5494746624
TYMP	2.2430083638
UBE2L6	3.9246935252
USP18	6.1108370034
USP6NL	-2.398462694207281
VAT1L	2.9662084775
VEGFA	4.018661329
VIM	2.7419647038
WASF2	2.0960454498
WLS	2.3831999526
WSB1	2.0540979807
XAF1	4.8659786827
ZBED3	2.5084479454
ZNF462	2.6814809424
ZNF506	-2.326046599800328
ZNF99	-2.6340997871033363
