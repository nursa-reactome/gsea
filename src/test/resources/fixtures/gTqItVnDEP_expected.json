[
    {"pathway": "CYTOKINE SIGNALING IN IMMUNE SYSTEM", "score": 0.84429544, "regulationType": "UP"},
    {"pathway": "IMMUNE SYSTEM", "score": 0.81702095, "regulationType": "UP"},
    {"pathway": "INNATE IMMUNE SYSTEM", "score": 0.3725799, "regulationType": "UP"},
    {"pathway": "INTERFERON SIGNALING", "score": 0.8661911, "regulationType": "UP"},
    {"pathway": "INTERFERON ALPHA/BETA SIGNALING", "score": 0.87766594, "regulationType": "UP"},
    {"pathway": "METABOLISM", "score": 0.15260126, "regulationType": "UP"},
    {"pathway": "SIGNAL TRANSDUCTION", "score": 0.2226403, "regulationType": "UP"}
]
